teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(id, age, name, parent_permission: true)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Just as I suggested above, you need to accept the arguments in the initialize method before passing it down to super. I'll suggest you update this also. :+1: :100: